### ISSUE TITLE
Cannot save Submissions via a Console command

### ISSUE THREAD

**Issue by iainsaxon** [verbb/formie#565]
When I try to save a Submission in a console command it throws an error:

```
Exception 'yii\base\UnknownMethodException' with message 'Calling unknown method: craft\console\Request::getBodyParam()'
```

My solution was to add the following to the Form::getRelationsFromRequest() method:

```
diff --git a/src/elements/Form.php b/src/elements/Form.php
index 137ec9d3..f6ecd688 100644
--- a/src/elements/Form.php
+++ b/src/elements/Form.php
@@ -953,6 +953,11 @@ public function getRelations()
      */
     public function getRelationsFromRequest()
     {
+        // nothing to do here for console requests
+        if (Craft::$app->getRequest()->getIsConsoleRequest()) {
+            return null;
+        }
+
         $value = (string)Craft::$app->getRequest()->getBodyParam('relations', '');
 
         return Json::decode(StringHelper::decdec($value));
```

Are there any negative side effects to skipping the relations step for a console command?

Thanks :) .

**Additional info**
- Plugin version: 1.4.14
- Craft version: 3.7.10
- Multi-site: No

**Additional context**

The call to save the Submission element:

```
$submission->setFieldValue('dateNotificationSentAt', $nowDate);
Craft::$app->getElements()->saveElement($submission, false);
```

It does seem similar to #542 however when debugging the code it was specifically this method trying to get the post data that caused the error.

**Comment by engram-design** [comment]
Yep, skipping that for console requests is totally fine. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
